website/ko/05-hello-world.md

```diff
@@ -235,9 +238,9 @@ For hexadecimal numbers, we output from the most significant *nibble* (a hexadec
 void printf(const char *fmt, ...);
 ```
 
-We're simply defining these as aliases for the versions with `__builtin_` prefixed. They are builtin features provided by the compiler (clang) itself ([Reference: clang documentation](https://clang.llvm.org/docs/LanguageExtensions.html#variadic-function-builtins)). The compiler will handle the rest appropriately, so we don't need to worry about it.
+`__builtin_`으로 시작하는 식별자들은 컴파일러(예: clang)에서 제공하는 빌트인 기능입니다(참고: [Clang 문서]((https://clang.llvm.org/docs/LanguageExtensions.html#variadic-function-builtins))). 내부 처리는 컴파일러가 알아서 해주므로, 우리는 단순히 이러한 매크로를 정의만 해두면 됩니다.
```

Comment:
nitpick: VitePress interprets this URL as `(https://...`:

<img width="841" alt="Screenshot 2025-01-23 at 10 11 35" src="https://github.com/user-attachments/assets/86f0000f-62ce-4995-96fa-1da2fe8d1c89" />

```suggestion
`__builtin_`으로 시작하는 식별자들은 컴파일러(예: clang)에서 제공하는 빌트인 기능입니다(참고: [Clang 문서](https://clang.llvm.org/docs/LanguageExtensions.html#variadic-function-builtins)). 내부 처리는 컴파일러가 알아서 해주므로, 우리는 단순히 이러한 매크로를 정의만 해두면 됩니다.
```